ReSharper.FSharp/src/FSharp.Psi.Features/src/Parsing/FSharpTreeBuilderBase.fs

```diff
@@ -400,33 +370,26 @@ type FSharpTreeBuilderBase(lexer, document: IDocument, lifetime, path: VirtualFi
             let representationMark = x.Mark(range)
 
             if not fields.IsEmpty then
-                let (SynField(range = firstFieldRange)) as firstField = fields.Head
```

Comment:
Likewise, please don't fix this warning. 🙂
I'll likely allow this case in the analyzer.